platforms/quorum/releases/dev/carrier-ns/namespace.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
```

Comment:
do not check-in contents of releases folder. Remove "dev" before each pull request